web/src/global.d.ts

```diff
@@ -6,8 +6,26 @@
 declare let zulip_test: any; // eslint-disable-line @typescript-eslint/no-explicit-any
 
 interface JQuery {
+    carousel: (
+        option?:
+            | "prev"
+            | "next"
+            | "pause"
+            | "cycle"
+            | number
+            | {
+                  [key: string]: unknown;
+                  interval?: number | false;
+                  pause?: "hover" | false;
+              },
+    ) => void;
```

Comment:
Just a note that we're planning to remove the one Bootstrap carousel in [our /hello redesign](https://github.com/zulip/zulip/pull/24301); we should at least add a comment here that this can be removed soon.

Reply:
Okay